easybuild/tools/modules.py

```diff
@@ -48,12 +48,15 @@
 from easybuild.tools.build_log import EasyBuildError
 from easybuild.tools.config import build_option, get_modules_tool, install_path
 from easybuild.tools.environment import restore_env
-from easybuild.tools.filetools import convert_name, mkdir, read_file, path_matches, which
+from easybuild.tools.filetools import convert_name, copy_file, mkdir, read_file, path_matches, which, write_file
 from easybuild.tools.module_naming_scheme import DEVEL_MODULE_SUFFIX
 from easybuild.tools.run import run_cmd
 from easybuild.tools.toolchain import DUMMY_TOOLCHAIN_NAME, DUMMY_TOOLCHAIN_VERSION
 from vsc.utils.missing import nub
 
+# relative to user's home directory
+LMOD_USER_CACHE_RELDIR = '.lmod.d/.cache'
```

Comment:
this is just a default, `--update-lmod-caches` takes an argument if you want to update caches in another location

since `.lmod.d/.cache` is the default in Lmod, I think just keeping this as is makes sense